build_automation/content_management/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url, include
```

Comment:
Since the views are not ready for this app yet, how about removing this file, or just removing the entries in urlpatterns?